### ISSUE TITLE
Remove MvxColor definitions

### ISSUE THREAD

**Issue by arturs105** [toggl/mobileapp#4186]
Once #4181 is merged, we can start moving away from `MvxColor` as a part of MvExit.

This issue is for the long file, that contains the color definitions - `Toggl.Foundation.MvvmCross.Helper.Color`.

We now have the option to move all the defined colors in the `Multiac.Color` struct, but i think the color definitions should remain in a separate file, so the color struct doesn't get too bloated and is easier to read.

Furthermore, i think we should rename the file with definitions to plural `Colors`, because currently we have 3 files named `Color.cs` and file searching is confusing because of that. There will be a separate issue to remove the third `Color.cs`, so in the end we will only have `Color.cs` and `Colors.cs`.

**Comment by heytherewill** [comment]
I am a bit against this one. I agree that it should be named `Colors`, but I don't know if it should live in Toggl.Foundation when color is clearly a UI related concept.
